contrib/!lang/ruby/packages.el

```diff
@@ -32,6 +32,14 @@
     :config (add-hook 'enh-ruby-mode-hook
                       (lambda () (rbenv-use-corresponding)))))
 
+(defun ruby/init-chruby ()
+  "Initialize chruby mode"
+  (use-package chruby
+    :defer t
+    :init (chruby)
+    :config (add-hook 'enh-ruby-mode-hook
+                      (lambda () (chruby-use-corresponding)))))
```

Comment:
You don't need the lambda expression here you can replace it by `'chruby-use-corresponding`. The lambda expression (or a named function) is needed only when you have to pass arguments to the function to hook.